src/test/java/com/github/britter/junit/jupiter/testkit/TestkitExamplesTests.java

```diff
@@ -0,0 +1,94 @@
+package com.github.britter.junit.jupiter.testkit;
+
+import com.github.britter.junit.jupiter.testkit.TestExecutionStatistics.Statistics;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.github.britter.junit.jupiter.testkit.ExecutedElementType.CONTAINER;
+import static com.github.britter.junit.jupiter.testkit.ExecutedElementType.TEST;
+import static com.github.britter.junit.jupiter.testkit.ExecutionEvent.Type.SKIPPED;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.platform.engine.TestExecutionResult.Status.FAILED;
+
+class TestkitExamplesTests {
+
+    @ExtensionTest(InnerTestCase.class)
```

Comment:
I think the goal here is that we want to have the testkit have names that aren't tied specifically do the concept of an Extension.  There are usages around meta testing parts of the junit modules themselves (not even related to Extensions) where the testkit is used, so having the name `ExtensionTest` seems a bit misleading?

Reply:
You're write. I talked to @sormuras about this and the plan is to do implement this jupiter API in a separate project, maybe in the junit-pioneer organization